Makefile

```diff
@@ -54,11 +56,8 @@ clean:
 # Building the binary
 ###############################################################################
 # Build the vendor directory.
-vendor: glide.lock
-	# To build without Docker just run "glide install -strip-vendor"
-	# Ensure that the glide cache directory exists.
-	mkdir -p $(HOME)/.glide
-	$(DOCKER_GO_BUILD) glide install --strip-vendor
+vendor: go.mod
```

Comment:
Probably want to rely on `go.sum` here, since we populate the vendor directory from the sum file.